### PR TITLE
CAMEL-13699 Add Json to Json transformation using JSLT

### PR DESCRIPTION
**Pull request by JiriOndrusek**
Issue: https://issues.apache.org/jira/browse/CAMEL-13699

New component for JSLT added.

### CODE REVIEW

File: components/camel-jslt/src/test/resources/log4j2.properties [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+## Unless required by applicable law or agreed to in writing, software
+## distributed under the License is distributed on an "AS IS" BASIS,
+## WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+## See the License for the specific language governing permissions and
+## limitations under the License.
+## ---------------------------------------------------------------------------
+
+appender.file.type = File
+appender.file.name = file
+appender.file.fileName = target/camel-jolt-test.log
```
Comment (davsclaus) [inline components/camel-jslt/src/test/resources/log4j2.properties]: should be camel-jslt-test

Comment (JiriOndrusek) [reply components/camel-jslt/src/test/resources/log4j2.properties]: Thx, I've missed that one. Fixed.

**Comment by oscerd** [comment]
I'll merge this ASAP

**Comment by oscerd** [comment]
Merged on master.

**Comment by oscerd** [comment]
Thanks

**Pull request closed without merging**